src/main/java/org/gooru/insights/services/ValidateUserPermissionServiceImpl.java

```diff
@@ -210,4 +211,8 @@ public String getAllowedParties(String traceId,RequestParamsDTO requestParamsDTO
 		return getRoleBasedParty(traceId,partyPermissions, allowedParty);
 	}
 
+	public BusinessLogicService getBusinessLogicService() {
+		return businessLogicService;
```

Comment:
BusinessLogicService

Can we re-factor the service class  with appropriate name ?